api/top-langs.js

```diff
@@ -9,7 +9,7 @@ import {
   renderError,
 } from "../src/common/utils.js";
 import { fetchTopLanguages } from "../src/fetchers/top-languages-fetcher.js";
-import { isLocaleAvailable } from "../src/translations.js";
+import { isLocaleAvailable } from "../src/assets/translations.js";
```

Comment:
translations are not an asset though. Would've put it on assets if we had .json locale files but for this it also exports utilities and the locale so let's keep it on the root itself.

Reply:
Done 👍.